modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/tests/TestrayProjectsAdministrator.testcase

```diff
@@ -129,11 +129,23 @@ definition {
 	}
 
 	@description = "This is a test for LPS-160541. Verify if as an Administrator I should not be able to add a duplicate Project"
-	@ignore = "Test Stub"
-	@priority = "3"
+	@priority = "5"
 	test AdminCanNotAddDuplicateProject {
+		task ("Go to Testray, Create Project duplicate") {
```

Comment:
Hi @joelrocha68, just a quick fix: instead of 'Go to Testray, Create Project duplicate' you can use 'Go to Testray and create a duplicated project'